ob_v3p0/context.json

```
@@ -3,26 +3,88 @@
         "id" : "@id",
         "type" : "@type",
 
-        "obi" : "https://purl.imsglobal.org/spec/ob/vocab#",
+        "obi" : "https://w3id.org/openbadges#",
+        "clr": "https://purl.imsglobal.org/spec/clr/v1p0/context/clr_v1p0.html",
+        "cred": "https://www.w3.org/2018/credentials/#",
+        "dc": "http://purl.org/dc/terms/",
+        "schema": "http://schema.org/",
+        "sec": "https://w3id.org/security#",
         "xsd" : "http://www.w3.org/2001/XMLSchema#",
 
-        "assertion" : "obi:dtAssertion",
+        "Achievement": "clr:Achievement",
+        "Criteria": "obi:Criteria",
+        "Endorsement": "cred:VerifiableCredential",
+        "Evidence": "obi:Evidence",
+        "Extension": "obi:Extension",
+        "FrameValidation": "obi:FrameValidation",
+        "IdentityObject": "obi:IdentityObject",
+        "Image": "obi:Image",
+        "Profile": "obi:Profile",
```

If at all possible, let's avoid using prefixes in our context, especially in the property values.
So, instead of `obi:Profile`, full URL `https://w3id.org/openbadges#Profile`.
Although prefixes are commonly used in the RDF world pre-JSON-LD, they've turned out to be way more trouble down the line, if they're used in `@context`s.

```suggestion
        "Profile": "https://w3id.org/openbadges#Profile",
```